ts/screens/onboarding/components/NotificationsPreferencesPreview.tsx

```diff
@@ -1,65 +1,14 @@
 import React from "react";
-import { Image, View, StyleSheet } from "react-native";
-import NotificationsBackground from "../../../../img/onboarding/notifications_background.svg";
-import NotificationsBackgroundBlue from "../../../../img/onboarding/notifications_background_blue.svg";
-import { H4 } from "../../../components/core/typography/H4";
-import { H5 } from "../../../components/core/typography/H5";
-import { IOColors } from "../../../components/core/variables/IOColors";
-import customVariables from "../../../theme/variables";
-import I18n from "../../../i18n";
-import { TranslationKeys } from "../../../../locales/locales";
-
-const backgroundImageHeight = 200;
-const notificationHeight = 72;
+import { StyleSheet, ImageBackground } from "react-native";
+import NotificationBlueBg from "../../../../img/onboarding/notification_blue.png";
+import NotificationWhiteBg from "../../../../img/onboarding/notification_white.png";
+import { NotificationPreviewSample } from "./NotificationPreviewSample";
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
-    backgroundColor: IOColors.white,
-    alignItems: "center",
-    height: backgroundImageHeight
-  },
-  blue: {
-    backgroundColor: IOColors.blue
-  },
-  overlayContainer: {
-    position: "absolute",
-    top: 0,
-    bottom: 0,
-    left: 0,
-    right: 0,
-    alignItems: "center",
-    justifyContent: "flex-end"
-  },
-  overlay: {
-    width: "90%",
-    minHeight: notificationHeight,
-    backgroundColor: IOColors.white,
-    borderRadius: customVariables.borderRadiusBase,
-    borderWidth: 1,
-    borderColor: IOColors.bluegreyLight,
-    flexDirection: "row",
-    alignItems: "center",
-    justifyContent: "flex-start"
-  },
-
-  previewNotificationMargin: {
-    bottom: 60
-  },
-
-  overlayWhiteBg: {
-    backgroundColor: IOColors.white // Modify the background color for the white background style
-  },
-  overlayImage: {
-    width: 25,
-    height: 25,
-    marginRight: "5%",
-    marginLeft: "3%",
-    borderRadius: customVariables.borderRadiusBase
-  },
-
-  overlayText: {
-    flexDirection: "column"
+    display: "flex",
```

Comment:
This can be omitted since in react-native flex should be the default value 😄

Reply:
Changed here https://github.com/pagopa/io-app/pull/4911/commits/4f5f0ff483d4f99ce2734221d99b6ab918be629c